README.md

```diff
@@ -106,7 +106,7 @@ I would like to thank all my coding partners who helped me in understanding lint
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](MIT) licensed.
+This project is [MIT](https://github.com/mumo-esther/setting-up-linters/blob/main/LICENSE) licensed.
 
 _NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```

Comment:
- [ ] Please kindly add at least one or two key features and future features sections to the README file to make it more professional. ℹ️